allocate.go

```diff
@@ -337,6 +342,15 @@ func Flag(name string, value interface{}) ExecAllocatorOption {
 	}
 }
 
+// Env is a generic environment variable in the form NAME=value to pass into
+// the new Chrome process. These will be appended to the environment of the
+// golang process as retrieved by os.Environ().
+func Env(variable string) ExecAllocatorOption {
```

Comment:
This might as well just take `...string`, so that setting multiple vars doesn't require multiple options.